docs/tags/common-schema.md

```diff
@@ -203,6 +203,21 @@ Field Name | Field Type | Required | Notes
 ```creator``` | string       | no       | Person, application, or component that created an event. E.g. ```John Smith```, ```Elasticsearch```, ```Some Batch Job```
 ```data``` | string       | no       | Additional event data. It can be anything you may find useful to have along inside of event object. E.g., it could be  stacktrace in case of ```app_error``` event, base64 encoded content of file, etc.
 
+### Kubernetes Event Tags
+Kubernetes events represent a special type of events that show what's happening inside a cluster, node, pod, or container.
```

Comment:
Kubernetes events represent a special type of events that show what's happening inside a cluster, node, pod, or container.
=> 
Kubernetes events show what's happening inside a cluster, node, pod, or container.

Reply:
Fixed